data-share/pom.xml

```diff
@@ -27,8 +27,8 @@
 
         <spring-cloud-config.version>2.0.4.RELEASE</spring-cloud-config.version>
         <h2.version>1.4.197</h2.version>
-        <kernel.core.version>1.3.0-SNAPSHOT</kernel.core.version>
-        <kernel.logger.logback.version>1.3.0-SNAPSHOT</kernel.logger.logback.version>
+        <kernel.core.version>1.3.0-beta.1</kernel.core.version>
+        <kernel.logger.logback.version>1.3.0-beta.1</kernel.logger.logback.version>
         <kernel.crypto-jce.version>1.0.6</kernel.crypto-jce.version>

```

Comment:
Are we using this:
<kernel.crypto-jce.version>1.0.6</kernel.crypto-jce.version>

Reply:
not using this property. hence removing it.